src/runtime_config/default/actions.js

```diff
@@ -829,4 +829,25 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "remove_node_2tx",
```

Comment:
I would like to avoid having a separate gov action here. Is there any way I can find out whether we're in two-tx or BFT mode so I can make "remove_node" dependent on that? Should we perhaps save that information in the service config table?

Reply:
That's an excellent idea, we can do the same thing for adding nodes. The service table is an excellent place for the flag.

Reply:
Done!